clients/cli/build/release.sh

```diff
@@ -15,13 +15,20 @@ sed -e "s/VERSION/${VERSION}/g" ./build/innosetup/phrase-cli.iss.template > ./bu
 
 IMAGE_PREFIX=phrase/phrase-cli
 IMAGE=${IMAGE_PREFIX}:${VERSION}
+IMAGE_LATEST=${IMAGE_PREFIX}:latest
 
 echo build docker image ${IMAGE}
 docker build --tag ${IMAGE} -f ./Dockerfile .
 
 echo push image ${IMAGE}
 docker push ${IMAGE}
 
+echo build docker image ${IMAGE_LATEST}
+docker build --tag ${IMAGE_LATEST} -f ./Dockerfile .
```

Comment:
Do we need to build and push the image twice?

We should rather change the original to be:

```bash
$ docker build --tag ${IMAGE} --tag ${IMAGE_LATEST} -f ./Dockerfile .
```